### ISSUE TITLE
The song cannot be resumed after being paused.

### ISSUE THREAD

**Issue by SmitMewada** [KushGabani/radiokey#13]
n a scenario where a song is paused by the bot, The song does not resume using the `*resume` or `*res` command. It seems like a discord-player package issue as all the other edge cases are executed in their respective scenarios.
The code for the resume command is in `src/actions/resume.js`

**Comment by KushGabani** [comment]
Yes, you seem to be right here. I just checked the bot in this specific scenario and it doesn't resume. It seems to be a package issue. maybe with some future updates, it can be fixed. But if still, someone thinks they can fix this issue by diving into `node_modules` then they are most welcome.
